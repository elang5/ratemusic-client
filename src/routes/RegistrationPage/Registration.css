.reg-page {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  background: url('https://images.unsplash.com/3/www.madebyvadim.com.jpg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1661&q=80') center;
  background-size: cover;
  background-repeat: no-repeat;
}