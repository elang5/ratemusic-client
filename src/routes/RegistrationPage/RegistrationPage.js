import React, { Component } from 'react'
import RegistrationForm from '../../components/RegistrationForm/RegistrationForm'
import Registration from './Registration.css'

export class RegistrationPage extends Component {
  static defaultProps = {
    history: {
      push: () => {}
    }
  }

  handleRegistrationSucces = () => {
    const { history } = this.props
    history.push('/login')
  }
  render() {
    return (
      <section className="reg-page">
        <RegistrationForm
          onRegistrationSuccess={this.handleRegistrationSucces}
        />
      </section>
    )
  }
}

export default RegistrationPage
