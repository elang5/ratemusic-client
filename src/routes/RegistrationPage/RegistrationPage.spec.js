import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import RegistrationPage from './RegistrationPage'

describe(`RegistrationPage Component`, () => {
  it(`renders the RegistrationPage component`, () => {
    const wrapper = shallow(<RegistrationPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})