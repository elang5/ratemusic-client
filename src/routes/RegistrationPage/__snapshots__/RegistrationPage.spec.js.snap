// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationPage Component renders the RegistrationPage component 1`] = `
<section
  className="reg-page"
>
  <RegistrationForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;
